sites/gcweb-menu/_base.scss

```
@@ -141,7 +141,7 @@
 		color: #000;
 		left:360px;
 		margin-bottom: 25px;
-		min-height: 815px;
+		min-height: 880px;
 		padding: 0px 39px 24px 39px;
 		top: 0;
 		width: 810px;
```

Please adjust the height of the menu in French as the list is taller.
```suggestion
		width: 810px;
		
		[lang=fr] & {
			min-height: 931px;
		}
```